src/languages/es.js

```diff
@@ -148,6 +148,8 @@ export default {
         someone: 'Alguien',
         total: 'Total',
         edit: 'Editar',
+        showMore: 'Mostrar más',
+        merchant: 'comerciante',
```

Comment:
Is this correct? Just asking because first letter is not capitalized like English

Reply:
Yep, should be capitalized.